src/bundle/Resources/public/js/scripts/admin.distraction.free.mode.js

```diff
@@ -16,6 +17,43 @@
         activeFieldEdit.classList.toggle('ibexa-field-edit--distraction-free-mode-active', active);
         editorInstance.set('distractionFreeModeActive', active);
 
+        if (active) {
```

Comment:
Should we safeguard the possibility of calling `changeDistractionFreeModeState` two times in a row with the same _active_ parameter value by e.g. checking whether `clearedPositionNodesData` is not empty?

Reply:
I have analyzed this part, and I think it's more foolproof is without any extra condition, as it will assure correct placement on each run - so it's immune to changing positions/overflow below when distraction free mode is on.

The only "problem" is that it has to traverse whole DOM tree on rerunning it with true, but I think it's small price to pay.